usaspending_api/disaster/management/sql/spark/disaster_covid19_file_d2_awards.py

```diff
@@ -0,0 +1,199 @@
+d2_awards_sql_string = """
```

Comment:
Why break from the previous style of having these in .sql files and change to .py files? SQL will add some benefit of syntax highlighting in IDEs. There is an extra step to getting non .py files to show up on Databricks when pip installing the repo; I can't recall if we added it or not, but had to add something maybe to the setup.py or the pyproject.toml to ensure all files, not just python files got installed with pip install.

Reply:
> There is an extra step to getting non .py files to show up on Databricks when pip installing the repo;

This is exactly why we are storing the sql in python modules. The existing pattern in our code base is to store spark sql in variables. SQL files from our code base is not uploaded or accessible when in Databricks. Instead of establishing a new pattern we opted to follow the existing pattern.

Reply:
I agree with Desmond here. All Spark SQL being run in Databricks is currently stored in .py files.